pagopa/io-web-profile
# [IOPID-702] : Feature flag for identity list

## Short description

Feature flag to hide identity list until the API is created in future versions

## List of changes proposed in this pull request

- Flag in .env files to hide idp list

## How to test

1. Check in .env files that NEXT_PUBLIC_FEATURE_FLAG is set to false
2. Login
3. Check in pages /blocco-accesso and /blocco-accesso/operazione-completata the IDP list is hiden
